src/dosbox.cpp

```diff
@@ -1135,6 +1135,11 @@ void DOSBOX_Init()
 	        "(auto by default, enabled if DOS version >= 7.0).\n"
 	        "FreeDOS and MS-DOS 7/8 COMMAND.COM supports this behavior.");
 
+	pbool = secprop->Add_bool("shell_config_shortcut", when_idle, true);
+	pbool->Set_help("Enable shell shortcuts for easy configuration changing, i.e. instead of\n"
+	                "'config -set property value' it is enough to execute 'config value' command\n"
+	                "(enabled by default).\n");
+
```

Comment:
The description is good, but need to be extended because querying config settings is also possible by executing their name only.

Reply:
The default value usually goes on top, at the end of the short summary, so I'd mention it before the "i.e." part, then from "i.e." I'd turn that into a new sentence (that's the detailed description).

Reply:
Don't you mean "it is enough to execute 'property value'"?

Reply:
Yeah, "setting" instead of "property" or "property value" should be used, actually. That's the name I'm consistently using in the manual ("setting", or "config setting").

Property is too ambiguous and/or too technical; setting or config setting for disambiguation is a good term understood by everybody, so let's use that for consistency (I've already normalised all config descriptions to only use this term).

Reply:
Suggest a common example instead of placeholders:

The shell config shortcuts allow directly making conf changes, i.e.: `C:\>sbtype sb16` without using the CONFIG tool: `C:\> config -set sblaster sbtype = sb16`

Reply:
> I think using "config" can be confused with the "config" command.

No, because you talk about "settings", "config settings", or "the `CONFIG` command". I will set up a glossary at some point for the manual so everybody follows the same terms consistently. Examples:

"You can change the `glshader` config setting to <blah blah>. What this setting does is <more blah>"
"Run the `CONFIG` command to <...>"

It's 100% unambigous; the key is consistency 😄 Property may be familiar to us devs, but it's a super weird term to use in a manual meant for regular people of power users.

Reply:
> Suggest a common example instead of placeholders:
> 
> The shortcuts allow conf settings to be changed directly, for example: `C:\>sbtype sb16`, instead of using the more combersome CONFIG tool, such as: `C:\>config -set sblaster sbtype = sb16`

Even better! 🚀 Writing concisely and clearly is really hard; usually my 4th or 5th iteration is that I keep for config descriptions, and let's not even get into the user manual situation 😄

Reply:
Thanks @johnnovak :-)

At least everyone mucking with conf settings knows about `sbtype`; it's been there since the first release and will be there forever more. Another commonly used one is `cycles` as well; whatever you like @FeralChild64.

And just for fun, this setting itself can be set via shortcut, so it could be described as: 

> Shortcuts allow conf settings to be changed directly, for example: `C:\>allow_shell_config_shortcuts no` will disable shell config shortcuts, after which the more cumbersome CONFIG tool would be needed re-enable them: `C:\>config -set dosbox allow_shell_config_shortcuts = yes`.

But that makes my head spin :face_with_spiral_eyes: :laughing:

Reply:
Uhuhh, that sentence is too complicated 😅 A bit baroque 😎 Your first attempt is much better and conveys all the user needs to know perfectly and succinctly:

> The shortcuts allow conf settings to be changed directly, for example: `C:\>sbtype sb16`, instead of using the more cumbersome CONFIG tool, such as: `C:\>config -set sblaster sbtype = sb16`

(And nice touch including `C:\>` at the start of the commands to make it clear it's about executing stuff on the command line. These little touches go a long way towards removing ambiguity and confusion).

Reply:
> Uhuhh, that sentence is too complicated

(lol.. yeah; I was just joking around w/ that one! :rofl: )

Reply:
After reading the whole discussion I have rewritten the description. Feel free to reopen if you have more remarks.